### PR TITLE
Add rich descriptions for automation triggers when viewing an automation

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Adds a new description component for cloud and oss automations to implement. Which will render a rich description for any trigger template. 

## Deployment status
**Before**
image
**After**
image

## Flow run state
**Before**
image
**After**
image

## Work pool status
**Before**
image
**After**
image

## Work queue status
**Before**
image
**After**
image

## Custom
**Before**
image
**After**
image

### CODE REVIEW

File: src/compositions/useWorkPools.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { useWorkspaceApi } from '@/compositions/useWorkspaceApi'
+import { WorkPool, WorkPoolsFilter } from '@/models'
+import { WorkspaceWorkPoolsApi } from '@/services'
+
+type UseWorkPools = {
+  subscription: UseSubscription<WorkspaceWorkPoolsApi['getWorkPools']>,
+  workPools: ComputedRef<WorkPool[]>,
+}
+
+export function useWorkPools(filter: MaybeRefOrGetter<WorkPoolsFilter | null | undefined>, options?: SubscriptionOptions): UseWorkPools {
```
Comment (collincchoy) [inline src/compositions/useWorkPools.ts:15]: 🙏

File: src/maps/filters.ts [hunk 2]
```diff
@@ -511,7 +511,7 @@ export const mapWorkQueuesFilter: MapFunction<WorkQueuesFilter, WorkQueuesFilter
   return removeEmptyObjects({
     work_queues: {
       ...toOperator(source.workQueues?.operator),
-      id: toAny(source.workQueues?.name),
```
Comment (collincchoy) [inline src/maps/filters.ts:514]: 🤨

**Review by collincchoy — APPROVED** [review]

File: src/maps/filters.ts (hunk 2 above)
Comment (pleek91) [reply src/maps/filters.ts:514]: Best I can tell this never worked

Comment (collincchoy) [reply src/maps/filters.ts:514]: 🙈

**Pull request merged**